README.md

```diff
@@ -71,3 +71,34 @@ Fork this repo and send us a pull request when you think it's ready for review.
 > What if I have a question?
 
 Create a new issue in the repo and we will get back to you shortly.
+
+
+## Installation
+
+```bash
+$ yarn install
+```
+
+## Running the app
+```bash
+# development
+$ npm run start
+
+# watch mode
+$ npm run webpack
+then 
+$ npm run start:dev 
+in new Terminal
+
+# production mode
+$ npm run start:prod
+```
+
+## Running test case
+```bash
+$ npm run test
+```
+## Swagger UI
+```
+http://localhost:3000/api
```

Comment:
I have another project running on 3000, after I launched this server how do I access it? I don't see logs in console.

Reply:
the application start success when the console log "Nest application successfully started "。

Reply:
OK